src/activities/imageRecognition/ImageRecognition.jsx

```diff
@@ -190,6 +208,17 @@ module.exports = class ImageRecognition extends React.Component {
       }
       {
         this.state.activityState === ActivityState.Playing &&
+        <div>
+        <Row>
```

Comment:
it looks like the formatting here and above is wonky -- mind if i pull your branch and use my editor's formatting tool on it? it's probably easier than me nitpicking indentation 😂

Reply:
Fine by me. I was going to merge this then run Prettier on everything. I resisted doing that here because it changes _a lot_ of lines.

Reply:
ohh, no, that's a great idea -- let's just do it all at once

Reply:
I should've done it first but by the time I thought of it I was in too deep. 😂 

I'll fix the (I assume) lint errors from the build though.